packages/helix/project.bri

```diff
@@ -0,0 +1,48 @@
+import * as std from "std";
+import git from "git";
+import rust, { cargoBuild } from "rust";
+
+export const project = {
+  name: "helix",
+  version: "25.01.1",
+  repository: "https://github.com/helix-editor/helix.git",
+};
+
+const source = Brioche.gitCheckout({
+  repository: project.repository,
+  ref: project.version,
+});
+
+export default function helix(): std.Recipe<std.Directory> {
+  return cargoBuild({
+    env: {
+      HELIX_DISABLE_AUTO_GRAMMAR_BUILD: "1",
+    },
+    path: "helix-term",
+    runnable: "hx",
+    source,
+  });
+}
+
+export async function test(): Promise<std.Recipe<std.File>> {
+  const script = std.runBash`
+    helix --version | tee "$BRIOCHE_OUTPUT"
+  `
+    .dependencies(rust, helix)
+    .toFile();
+
+  const result = (await script.read()).trim();
+
+  // Check that the result contains the expected version
+  const expected = `helix ${project.version}`;
```

Comment:
I saw that `hx --version` outputs: `helix 25.01.1 (e7ac2fcd)`, if you want to do an exact match with the short git commit hash, you could look at the package `proot`, we are using a trick there to check the short git commit hash in the test method.